.python-version

```diff
@@ -1 +1 @@
-3.8.13
+3.10.5
```

Comment:
Note that we'll need to ensure this version of Python is present on our build agents: https://github.com/grapl-security/grapl-infrastructure/blob/e9e86c60b18fd5f152d87c75d19079847f65b784/packer/buildkite-base-ami.pkr.hcl#L197-L205